databuilder/query_engines/base_sql.py

```diff
@@ -238,9 +238,9 @@ def get_static_value(self, value: Union[Scalar, tuple, list]) -> StaticValue:
         # their elements at runtime. See: https://bugs.python.org/issue46191
         if isinstance(value, (tuple, list)) and any(
             not isinstance(v, SCALAR_TYPES) for v in value
-        ):
+        ):  # pragma: no cover
             assert False, f"Unhandled type {value!r}"
-        return value
+        return value  # pragma: no cover
```

Comment:
Just for my own education, how come this line needs `no cover`? Isn't the return always executed here

Reply:
I added it without thinking 🤦‍♂️  Fixed and rebased now.